recipes-core/ctemplate/ctemplate_git.bb

```diff
@@ -0,0 +1,26 @@
+SUMMARY = "ctemplate"
```

Comment:
Is it native only package? If yes, please add -native to the name of the file (like ctemplate-native_git.bb)

Reply:
It is required for target also